ansible/roles/ocp4-workload-acm-app-policies-lab/tasks/remove_workload.yml

```diff
@@ -0,0 +1,62 @@
+---
+# Implement your Workload removal tasks here
+
+# 1. Ensure we're connected to the cluster
+# 2. Make a temporary dir for cloning the deploy repo
+# 3. Clone the deploy repository
+# 4. Run the uninstall script
+
+
+- name: Ensure there is connectivity to OCP cluster as system:admin
+  command: oc whoami
+  register: whoami
+
+- name: Fail if not system:admin
+  fail:
+    msg: "Not connected to OCP as system:admin"
+  when: whoami.stdout != "system:admin"
+
+- name: Ensure deploy repository is cloned (1/2) - Ensure directory is removed if it already exists
+  file:
+    path: "deploy"
+    state: absent
+  become: true
+
+- name: Ensure deploy repository is cloned (2/2) - Clone the repository
+  git:
+    repo: "{{ _acm_install_repo }}"
+    dest: "deploy"
+    version: "{{ _acm_install_repo_branch }}"
+
```

Comment:
Can you add and use a `_acm_install_repo_dest` variable for that ?

Do you really need 2 tasks for that ?
Did you try `force` from the git module ?
If you don't need git history, you can use also `depth: 1`


```yaml
- name: Ensure deploy repository is cloned (2/2) - Clone the repository
  git:
    repo: "{{ _acm_install_repo }}"
    dest: "{{ _acm_install_repo_dest }}"
    version: "{{ _acm_install_repo_branch }}"
    force: true
    depth: 1
```